instance/tests/models/test_openedx_instance.py

```diff
@@ -467,6 +467,10 @@ def test_get_load_balancer_configuration(self, mocks):
             backend_map, config, domain_names, instance.active_appserver.server.public_ip
         )
 
+        # Test configuration after terminating appserver
+        with patch('instance.openstack.get_server_public_address', return_value=None):
+            self.assertEqual(instance.get_load_balancer_configuration(), ([], []))
```

Comment:
@smarnach Could you split this into a dedicated test? It's easy to miss what is being tested when several different tests are concatenated in one.